packages/toolpad-core/src/SignInPage/SignInPage.tsx

```diff
@@ -239,388 +250,421 @@ function SignInPage(props: SignInPageProps) {
   );
 
   return (
-    <Container component="main" maxWidth="xs">
-      <Box
-        sx={{
-          marginTop: 8,
-          display: 'flex',
-          flexDirection: 'column',
-          alignItems: 'center',
-        }}
-      >
-        {branding?.logo ?? (
-          <Avatar sx={{ my: 1, mb: 2, bgcolor: 'primary.main' }}>
-            <LockOutlinedIcon />
-          </Avatar>
-        )}
+    <Box
+      sx={{
+        minHeight: '100vh',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        ...sx,
+      }}
+    >
+      <Container component="main" maxWidth="xs">
+        <Box
+          sx={{
+            marginTop: 8,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            bgcolor: 'background.paper',
+            borderRadius: 1,
+            p: 4,
+            border: '1px solid',
+            borderColor: (theme) => alpha(theme.palette.grey[400], 0.4),
```

Comment:
If you're using functions in properties more than once I guess the whole `sx` can be a function as the first argument should be the `theme`?